R doc 1570 cloud supp (#1052)
ravendb/docs: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown

> # RavenDB on the Cloud: Pricing, Payment and Billing

Reviewer: It feels to me like this page does not belong to the documentation, it could be found also in ravendb.net or in the portal, as an explanation page. @ppekrol @ayende ?

Reviewer 2: I think we do need pricing discussion in the docs. That is quite important for users

     - [On-Demand Payment](../cloud/cloud-pricing-payment-billing#on-demand-payment)  
     - [Yearly with No Upfront payment](../cloud/cloud-pricing-payment-billing#yearly-with-no-upfront-payment)  
     - [Yearly with an Upfront payment](../cloud/cloud-pricing-payment-billing#yearly-with-an-upfront-payment)  
     - [Viewing your current configuration price](../cloud/cloud-pricing-payment-billing#viewing-current-configuration-price)  
     - [Additional Expenses](../cloud/cloud-pricing-payment-billing#additional-expenses)  
  * [Payment](../cloud/cloud-pricing-payment-billing#payment)  
  * [Billing](../cloud/cloud-pricing-payment-billing#billing)  
    - [Charging Failures](../cloud/cloud-pricing-payment-billing#charging-failures)  

{NOTE/}

{WARNING: }
Prices and fees in screenshots that appear in this documentation are for illustration only.  
{WARNING/}

---

{PANEL: Pricing}

You can pay for your instances by one of three models:  

* **On-Demand Payment**  
* **Yearly with _no_ Upfront payment**  
* **Yearly _with_ an Upfront payment**  

---

####On-Demand Payment  

The on-demand model lets you pay for your instance **by the minute**, and come and go as you please.  

> We will measure your instance run-time **from rising to termination**, and charge you with **minute granularity**.  

Reviewer 2: ```suggestion
We will measure your instance run-time **from provisioning to termination**, and charge you with **minute granularity**.  
```

You may find this model useful if you look for flexibility.  You are not tied to a particular configuration 
and are free to shift between configurations at will. 

{NOTE: }
We charge on-demand **by the minute**, with a **minimum limit of one hour**.  
If you run an instance for 93 minutes, you will pay for exactly 93 minutes.  
If you run it for less than an hour, say 42 minutes, you will still be charged for a full hour worth to match the minimum limit.  
{NOTE/}

---

####Yearly Payment  

> We support two models of yearly payments: **Yearly with no upfront payment**, and **Yearly with an upfront payment**.  

Reviewer: ```suggestion
We provide two models of yearly payments: **Yearly with no upfront payment**, and **Yearly with an upfront payment**.  
```

Author: done (offer)

Both grant you discounts in relation to the on-demand model.  

**Be aware** however that once a yearly contract has been activated, it is charged fully to the end of the year 
and cannot be revoked, halted or changed.  

{NOTE: }

> * Terminating a product you've been paying for yearly, will **not** stop your monthly payments.  

Reviewer 2: ```suggestion
* Terminating a product with a yearly contact, will **not** stop your future payments.  
```

> * Upscaling or downscaling a product you're paying for yearly, will **not** revoke the yearly agreement.  

Reviewer 2: ```suggestion
* Upscaling or downscaling a product with a yearly contract will **not** revoke the yearly agreement.  
```

{NOTE/}

---

####Yearly with NO upfront payment  
Committing for a year and **paying monthly**, grants you a **5% discount**.  
![Yearly No Upfront](images/pricing-003-payment-models-yearly-no-upfront.png "Yearly No Upfront")

---

####Yearly WITH an upfront payment  
Committing for a year and **paying for it all in advance**, grants you a **10% discount**.  
![Yearly Upfront](images/pricing-004-payment-models-yearly-upfront.png "Yearly Upfront")

{NOTE: }
Yearly contracts are relevant only for [production instances](../cloud/cloud-instances#a-production-instance).  
[Development instances](../cloud/cloud-instances#a-development-instance) can only use the on-demand model.  
{NOTE/}

---

####Viewing current configuration price  
While [provisioning or editing](../cloud/portal/cloud-portal-products-tab) your product, 
costs are shown in the **Your Order** slot.  

The overall price is an outcome of the hardware components you chose and your pricing model, 
with possible [additional expenses](../cloud/cloud-pricing-payment-billing#additional-expenses).  

See for example the differences in price between the production configurations PB0 and PB1:  

> ![PB0](images/pricing_001_PB0.png "PB0")

Reviewer 2: PB10 , you meant?

Author: yup, thx

![PB1](images/pricing_002_PB1.png "PB1")
  
You can also find the cost in the [summary page] before actually running the instance, and always see it in your 
[Billing tab](../cloud/portal/cloud-portal-billing-tab).  

---

####Additional Expenses

Your pricing model does **not** cover incidental expenses over -  

* **Traffic Usage**  
  Charges for traffic to and from your instance will be added to your basic pricing plan.  

* **Expanding your Disk Storage**  
  We will automatically expand the amount of disk space allocated for your product when its usage reaches 90%, 

>   to prevent any chance of data loss. The new disk size will remain, and your payment plan will be updated accordingly.  

Reviewer 2: ```suggestion
  to avoid running out of disk space Your payment plan will be updated to reflect the new storage size.
```

Author: When the disk space allocated for your product is 90% full, we will expand it to avoid running out of storage.  
Your payment plan will be updated to reflect the new storage size.
